src/components/button/Button.tsx

```diff
@@ -0,0 +1,19 @@
+import { FC, ReactNode } from "react";
+import { ButtonVariantEnum } from "./Button.enum.ts";
+
+interface Props {
+  children: ReactNode;
+  variant: ButtonVariantEnum;
+  onClick?: () => void;
+}
+
+const Button: FC<Props> = ({ children, variant, onClick }) => {
+
+  return (
+    <button className={variant} onClick={() => onClick && onClick()}>
```

Comment:
HTMLButtonElement.onClick이 옵셔널하게 인자를 받기 때문에 &&로 검증할 필요가 없습니다.

```suggestion
    <button className={variant} onClick={onClick}>
```